src/ORM/FieldType/DBString.php

```diff
@@ -212,4 +215,28 @@ public function Plain()
     {
         return trim($this->RAW());
     }
+
+    /**
+     * Swap add for DefaultEllipsis if need be
+     * @param string $string
+     * @param false|string $add
+     * @return string
+     */
+    private function addEllipsis(string $string, $add): string
+    {
+        if ($add === false) {
+            $add = $this->DefaultEllipsis();
+        }
+
+        return $string . $add;
+    }
+
+    /**
+     * Get the default string to indicate that a string was cut off.
+     * @return string
+     */
+    public function DefaultEllipsis(): string
```

Comment:
```suggestion
    public function defaultEllipsis(): string
```

For PSR-12 compliance we should be aiming to stop naming methods with uppercase first letters

Reply:
done.